lib/nbv.js

```diff
@@ -183,6 +196,7 @@ var nbv = (function() {
 
                     dm = d.createElement('pre');
                     dm.style.margin = 0;
+                    dm.className = 'illustration-title'; // fixed: some outputs may be customized hidden, by nagexiucai
```

Comment:
What is this class referring to?

Reply:
Sorry, maybe i should comment it?

users can make 'display_data' hidden by using the 'illustration-title' like "display: none;", we konw.

Reply:
Sorry, maybe i should comment it?

users can make 'display_data' hidden by using the 'illustration-title' like "display: none;", we konw.

Reply:
I'd drop it, let the illustration title be seen.

Reply:
Okay, that is good, i can understand, the "illustration-title" is a temporary thing of mine, styles should be managed elsewhere.

Reply:
Okay, that is good, i can understand, the "illustration-title" is a temporary thing of mine, styles should be managed elsewhere.